media/js/mozorg/about-leadership.js

```diff
@@ -0,0 +1,108 @@
+            $origin = $origin.next().length ? $origin.next()
+                                            : $origin.siblings(':first');
+            personId = $origin.attr('id');
+        } else {
+            $origin = $origin.prev().length ? $origin.prev()
+                                            : $origin.siblings(':last');
+            personId = $origin.attr('id');
+        }
+
+        $('#modal').attr('aria-labelledby', personId).focus();
+        $('#modal .overlay-contents').replaceWith($origin.clone().attr('tabindex', '0').addClass('overlay-contents'));
+        $('#modal header h1').text($origin.find('.fn').text());
+
+        $origin.addClass('modal-origin');
+    };
+
+    // Set up the modal
+    $('.has-bio').each(function() {
+        var $this = $(this);
+        var modalTitle = '<h1>' + $this.find('.fn').text() + '</h1>';
+
+        $this.attr({
+            'tabindex': '0'
+        }).on('click', function() {
+
+            Mozilla.Modal.createModal(this, $this.clone().removeAttr('id'), {
+                title: modalTitle,
+                'onCreate': function() {
+                    var $nav = $('<nav role="presentation"></nav>').insertBefore('#modal-close');
+
+                    $('<button class="next" aria-controls="modal"></button>')
+                        .text(window.trans('global-next')).appendTo($nav);
+                    $('<button class="prev" aria-controls="modal">prev</button>')
+                        .text(window.trans('global-previous')).appendTo($nav);
+
+                    $nav.on('click', 'button', function() {
+                        var $this = $(this);
+
+                        navModal($this.hasClass('prev') ? -1 : 1);
+                    });
+                }
+            });
+
+        }).on('keydown', function(event) {
+            if (event.keyCode === 13) { // Enter
+                $this.trigger('click');
+            }
+        });
+
+    });
+
+    // Set up keyboard shortcuts for the modal
+    $(document).on('keydown', '#modal', function(event) {
+        var direction = 0;
+
+        switch (event.keyCode) {
+        case 37: // Left arrow
+            direction = ltr ? -1 : 1;
+            break;
+        case 38: // Up arrow
+            direction = -1;
+            break;
+        case 39: // Right arrow
+            direction = ltr ? 1 : -1;
+            break;
+        case 40: // Down arrow
+            direction = 1;
+            break;
+        }
+
+        if (direction) {
+            event.preventDefault();
+            navModal(direction);
+        }
+    });
+
+    // trigger modal on page load if hash is present and matches a person with a bio
+    if (window.location.hash) {
+        var $target = $(window.location.hash);
+
+        if ($target.length && $target.hasClass('vcard has-bio')) {
+            $target.trigger('click');
+        }
+    }
+
+    // open external links in new windows
+    $('a[rel="external"]').on('click', function(e) {
```

Comment:
This doesn't seem to be working, I'm guessing because of content is relocated within the modal. It should be noted however than opening links in new tabs is now considered to be both a security and performance issue. If we really need to do it, we should make sure to use `rel="external noopener noreferrer"`